### PR TITLE
Add prices daily

### PR DESCRIPTION
**Pull request by aalan3**
This PR aggregates prices into a daily table to be used for other daily tables, which hopefully should significantly help performance.

### CODE REVIEW

**Review by jeff-dude — APPROVED** [review]
good idea for daily spells, will be good to keep in mind each time those are built

will queue up to merge in a bit, as there is a large 1inch refresh happening now

File: models/prices/prices_usd_daily.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}}
+
+select * from (
+select
+    cast(date_trunc('day', minute) as date) as day,
+    blockchain,
+    contract_address,
+    decimals,
+    symbol,
+    price,
```
Comment (aalan3) [inline models/prices/prices_usd_daily.sql:20]: After discussing with @0xRobin we should use avg here instead

Comment (0xRobin) [reply models/prices/prices_usd_daily.sql]: update: added all of OHLC (open-high-low-close) price points to the schema

**Comment by jeff-dude** [comment]
what's the plan here?

**Comment by aalan3** [comment]
@jeff-dude the table looks good with the additions from @0xRobin.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**